src/app/components/mno-app-onboarding-btn/mno-app-onboarding-btn.html

```diff
@@ -9,8 +9,8 @@
     <span>{{$ctrl.conflictingApp.name}}</span>
   </div>
   <div ng-switch-when="INSTALLED_LAUNCH">
-    <i class="fa fa-check fa-2x button-check"></i><br>
-    <span class="text-muted">(connected)</span>
+    <span>No set-up required</span><br>
+    <span class="text-muted">This app connects when launched.</span>
```

Comment:
Please add those text string into locales keys.
(I forgot to i18n the `(connected)`)